### ISSUE TITLE
Allow to specify the scope for a PIM activation

### ISSUE THREAD

**Issue by breml** [netr0m/az-pim-cli#53]
### Welcome

- [X] Yes, I've searched similar [issues on GitHub](https://github.com/netr0m/az-pim-cli/issues) and didn't find any.

### Is your feature request related to a problem? Please describe

In my case in need to specify the scope of a PIM activation request.

### Describe the solution you'd like

Allow to specify the scope as command line flag.

### Describe alternatives you've considered

-

**Comment by netr0m** [comment]
Please elaborate. What do you mean by scope? Are you thinking of the "Justification" (i.e. `--reason` in the CLI for `az-pim-cli`)? For the activation request itself (`az-pim-cli activate`), the following fields are available in the API: https://github.com/netr0m/az-pim-cli/blob/e6d96fc864f0b8cb5bad5d53cceb6fa094937cd7/pkg/pim/models.go#L170-L180

**Comment by netr0m** [comment]
The `scope` is automatically selected, as this software (`az-pim-cli`) gets a list of roles and the subscription to which each belongs from the PIM API. The subscription is specified using either `--name <SUBSCRIPTION_NAME>` or `--prefix <PREFIX_OF_SUBSCRIPTION_NAME>`, and the role is specified using `--role <ROLE_NAME>`. The subscription (`--name` / `--prefix`) is always required. If `--role` is absent, the software will automatically select the first matching role in the list.

If you list roles, i.e. `az-pim-cli list`, you will get something like the following format:

```bash
$ az-pim-cli list
== SUBSCRIPTION_NAME_1 ==
         - ROLE_NAME_1
         - ROLE_NAME_2
== SUBSCRIPTION_NAME_2 ==
         - ROLE_NAME_1
```

for instance,
```bash
$ az-pim-cli list
== S100-Example-Subscription ==
         - Contributor
         - Owner
```
where you can activate the "Owner" role with `az-pim-cli --name S100-Example-Subscription --role Owner`, or the first match (here: "Contributor") with `az-pim-cli --name S100-Example-Subscription`.

As for the "Ticket system" and "Ticket number", those were recently implemented in #56 and is [currently being released](https://github.com/netr0m/az-pim-cli/actions/runs/10845769640/job/30097309632) as part of [v1.1.0](https://github.com/netr0m/az-pim-cli/releases/tag/v1.1.0)

The "reason" is already supported by using the `--reason <REASON>` flag.

**Comment by breml** [comment]
@netr0m Thanks for the explanation and the fix in #56 . I just tested it again and this time it worked as expected.
